compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__qualified_to_unqualified_import_nested_type_outer.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "\nimport option\nimport wobble\npub fn main(x) -> option.Option(wobble.Wibble) {\n    todo\n}\n"
+---
+----- BEFORE ACTION
+
+import option
+import wobble
+pub fn main(x) -> option.Option(wobble.Wibble) {
+           ↑                                    
```

Comment:
Why does this suggest the code action? The cursor is not within the type constructor.